### ISSUE TITLE
Improve pictures in Play-Store

### ISSUE THREAD

**Issue by Glitchy-Tozier** [florisboard/florisboard#1017]
Let me just go trough the pictures and share my thoughts with you.

**1&2**:  Fine. I'm a little bothered by the amount of empty space, but that's not a very pressing issue.
**3**: I think nobody cares about the number-layout. It's great that it exists but it's not a selling point.
**4**: Emojies. Good picture. (Same space-issue though)
**5-8**: I think non-open-source lovers don't have the AnySoftKeyboard-trauma we have. To most people, a picture of settings is pretty boring.

### Suggestions:
- [ ] Replace the number-layout image with a screenshot of the normal keyboard using a fun theme.
- [ ] If the settings-pictures are used to indicate certain functionality ("Hey,  we have glide typing"), replace them with screenshots of the functionality in action. (keyboard with a glide-trail and matching word-suggestions)
- [ ] I suggest somehow implementing screenshots showing the following features: Glide typing, word-suggestions, THEMES (!!!!).

**Comment by Hayleia** [comment]
Yeah I agree, they're not "bad" but they don't really sell the keyboard, which is a shame because it's great.

I'd also suggest to showcase language support. I dare anyone to find three keyboards that support both Korean and Swedish input. If somehow someone actually managed that (I mean, I even tried gboard and it did weird things instead of the intended things, see video below), do it again with the "open source" limitation.
https://user-images.githubusercontent.com/7904776/121766338-5b54b680-cb51-11eb-9369-2cfd76b1e845.mp4

Btw, they don't have to be screenshots, they can be collages too. Just like gboard showcases themes with several themes displayed on one picture here. This applies to language support too. Now of course it's more work, but we can probably discuss first then tone down "overkill" solutions later. It doesn't cost much to shoot for the moon first while we're only talking then do what's best realistically.
![image](https://user-images.githubusercontent.com/7904776/121766385-b2f32200-cb51-11eb-8361-88c2dd62b1c8.png)

And an easy collage would be for settings. I can definitely agree than several pictures of settings is wasting picture slots for something most people don't really want to see. But 1 picture with several screens of settings on it would only use 1 picture slot and would show efficiently that the keyboard is customizable to the user's liking.

**Comment by Glitchy-Tozier** [comment]
"The pictures don't sell the keyboard" is a great expression for what i thought about the current screenshots!

**Comment by patrickgold** [comment]
> "The pictures don't sell the keyboard" is a great expression for what i thought about the current screenshots!

This is indeed a good point, I must agree. The problem was that I didn't really had an idea for good pictures and I just took some screenshots. I will see what I can do for the v0.4.0 version.

**Comment by kj7rrv** [comment]
Once the arrow key layout comes back, should that have a screenshot? Being able to keep that from Gboard was a significant "selling point" for me, and TBH not having it has been a bit difficult at times.

**Comment by Glitchy-Tozier** [comment]
@kj7rrv In my opinion it's way less important (to the average user) than flashy themes, a good emoji-board, word suggestions, and glide typing.

Also, as far as I know Patrick plans on changing the layout a bit, so i'm not sure whether this selling-aspect will persist. (As it won't be a Gboard-copy anymore)

That being said, if there is enough space (which i'm not sure about as at least 5 screenshot-slots are taken), this might be one of the things to consider for pic. no. 6.

tl;dr: In my opinion, there are much more important shreenshots.

**Comment by patrickgold** [comment]
> In my opinion it's way less important (to the average user) than flashy themes, a good emoji-board, word suggestions, and glide typing.

This is true, especially for branding in the PlayStore (so many keyboards use the emojis, stickers, themes etc. as their screenshots, often times you don't really see the actual keyboard anymore but hundreds of emojis and other stuff).

I am not entirely sure if I should do this, but I could provide two sets of screenshots, one for the PlayStore with the "cool" stuff and one for F-Droid (the target audience of F-Droid is often times less interested in flashy things but in the core functionality and privacy).

As for the number of screenshots, I have 7 or 8 slots IIRC, but I think after a screenshot redesign 6 screenshots should be more than enough.

> Also, as far as I know Patrick plans on changing the layout a bit, so i'm not sure whether this selling-aspect will persist. (As it won't be a Gboard-copy anymore)

I could also provide two versions of that selection keyboard, the Gboard copy and a more advanced version. Shouldn't be too much of a hassle to support both IMO.

**Comment by ssssamstrup** [comment]
Sounds like a good idea with different pictures for each target audience. I'm on F-droid and definetly not looking for flashy stuff, though I love beautiful functional design and colours!

For the Play store version I think it's a good idea to focus:
1) on what features set florisboard apart from other keyboards
2) and show it has all the nice features you expect a normal keyboard to have.

**Comment by Glitchy-Tozier** [comment]
In any case (=in either of the stores) I think the privacy aspect is worth mentioning.

I suggest one screenshot somewhere in slot 3-4 that contains
- A big lock (IMPORTANT!!!!! 🙃)
- "No data connection"
- "Can't connect to the internet" (or a similar sentence)


Imo this should also be added to the Play-Store as many people may not constantly think of privacy, but still care about it when reminded of it's existence.

**Comment by Hayleia** [comment]
> I am not entirely sure if I should do this, but I could provide two sets of screenshots, one for the PlayStore with the "cool" stuff and one for F-Droid (the target audience of F-Droid is often times less interested in flashy things but in the core functionality and privacy).

There is also the perverse reverse side effect though. A keyboard on F-Droid is by default supposed to support core functionality and privacy (even more so with just the word "privacy" somewhere at the beginning of the description to confirm what people already expected, no screenshot needed). And on the other hand, a keyboard on F-Droid is by default supposed to have some trade-offs for privacy so not much "flashy stuff".

And conversely, if someone is looking for a keyboard on the Play Store and not on F-Droid, probably they would only see privacy as a bonus and they expect a keyboard to have glide typing and themes (if it doesn't, well there are tons of other keyboards on the Play Store...) and want confirmation of that before any mention of privacy.

------

Also not sure about you all but screenshots load slowly for me on F-Droid sometimes and they're at the bottom (after the description) so having like "🔒 No internet permission!!" at the beginning of the description would get the point about privacy way more efficiently than a screenshot with the same information.

And conversely, screenshots on the Play Store are at the top and load fast and I'm not even sure most people read the description.

------

So my conclusion is that in both cases, screenshots need to focus on the "flashy stuff"
- on the Play Store to quickly defend that Florisboard isn't inferior to others
- on F-Droid to set it apart from other keyboards which focus too much on privacy in their descriptions and seem like they made a lot of sacrifices to features

It doesn't change the fact that the description will be the first thing people see on F-Droid (once again at the top, especially visible with a lock emoji) and should be able to play the role confirm that privacy is a focus (and is actually the best place to do so), and then people can read the feature list and look at the screenshots at ease.

And nothing prevents the 4th screenshot on the Play Store to mention privacy as what sets it apart from other keyboards on the Play Store.

**Comment by ssssamstrup** [comment]
I think a lot of people on the Play store are also interested in Privacy, especially if people are reminded that keyboards *can/will* send your input to the developer, without their explicit permission.

So I think this should also be front and center on Play Store, since it's a unique selling point for Floris. 😊

**Comment by pslyman** [comment]
Just throwing this out there. There's a free PlayStore app called InstaMocks ([link](https://play.google.com/store/apps/details?id=com.instamocks&hl=en_US&gl=US)) that makes stylish screenshots easy. 
<img alt="stylized florisboard screenshot" src="https://user-images.githubusercontent.com/22920382/133474952-c94987f8-60e5-4373-9623-66ffe8c5b1ff.png" width="300"/>

I've used it a couple of times for apps I've published. I could whip up some more if you'd like, but I'd need some direction on what to include in the screenshots. From my experience, screenshots are intended to make the product more attractive, and offer a surface level view of the biggest selling points that can be captured.

**Comment by lm41** [comment]
The theme support could also be shown by something similar to [this image](https://raw.githubusercontent.com/SapuSeven/BetterUntis/master/fastlane/metadata/android/en-US/images/phoneScreenshots/2.png) from the betteruntis project

**Comment by lm41** [comment]
@Glitchy-Tozier Please have a look at the linked PR and give feedback.

**Comment by Glitchy-Tozier** [comment]
Great job, looks amazing! :)
The only thing I'd like to mention is that I find it difficult to distinguish the two darker themes on first glance. Due to this, it just seems like a comparison between two themes that's off-center.

I'd like to suggest adding a colorful theme as a third option instead. This could make the whole thing even more exciting :)
(For some reason I keep thinking a red theme may loog good there)
Thoughts?

**Comment by lm41** [comment]
@Glitchy-Tozier in the long run the screenshot generation should be done automatically so adding other themes shouldn't be an issue. I've added two cover images for the two tracks and think they are a good fit for FlorisBoard.
